src/every_query/aces_to_eq/aces_to_eq.py

```
@@ -0,0 +1,99 @@
+
+import hydra
+import polars as pl
+from hydra.utils import instantiate
+from omegaconf import DictConfig, OmegaConf
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def code_slug(code: str, n_hash: int = 10, prefix_len: int = 24) -> str:
+    h = hashlib.sha1(code.encode("utf-8")).hexdigest()[:n_hash]
```

Ruff (0.14.10)

Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)